.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: java
 jdk:
-  - openjdk7
+  - openjdk8
```

Comment:
Let's do the move to Java 8 in a separate PR, the `jdk.version` property in the pom.xml will need to change as well. We'd need to do a major version bump etc. To have Travis working for Java 7 you need to add
```
os:
  - linux
dist: trusty
```

Reply:
hmm travis doesn't seem to like this change, is there something else I need to add?